rpc/server.go

```diff
@@ -95,6 +95,7 @@ func NewThetaRPCServer(mempool *mempool.Mempool, ledger *ledger.Ledger, dispatch
 
 	t.server = &http.Server{
 		Handler: t.router,
+		IdleTimeout: viper.GetDuration(scom.CfgRPCTimeoutSecs)*time.Second,
```

Comment:
Should this line be changed to the following? i.e. use `CfgRPCIdleTimeoutSecs` instead of `CfgRPCTimeoutSecs`.

```IdleTimeout: viper.GetDuration(common.CfgRPCIdleTimeoutSecs) * time.Second,```